src/snapshot/SnapshotServer.cpp

```diff
@@ -163,28 +162,31 @@ SnapshotServer::recvPushSnapshotDiffs(const uint8_t* buffer, size_t bufferSize)
                 break;
             }
             case (faabric::util::SnapshotDataType::Int): {
-                const auto* value =
+                const auto* diffValue =
```

Comment:
Yes another good point, although the functions in `util/bytes.h` use `std::byte`, and without changing their signature to `uint8_t` in place of `std::byte` it looks like I can't use them directly on a `uint8_t` pointer. Given that everything else uses `uint8_t` I'd be inclined to change them to be in sync with the rest of the codebase, but would this break things for you?

Reply:
It's fine to change it, `std::byte` is fairly recent and meant to represent chunks of arbitrarily-typed memory as opposed to specifically arrays of 8-bit integers, they can be easily converted between each other (but not implicitly). I think using it to mean "block of memory" is useful, but it can be annoying to keep converting it back to uint8_t which most APIs expect - up to you which route you want to go.

Reply:
Ok cool, I think `std:byte` is the way to go, definitely easier to read, but perhaps a wholesale change in a single PR might be better than doing it bit by bit. I'll change them to `uint8_t` in this PR and add an issue.